### ISSUE TITLE
About Identifying Arbitrary Label Noise

### ISSUE THREAD

**Issue by txw1997** [thulas/dac-label-noise#8]
Thanks for your excellent work!
How can I use DAC for my own dataset which has noisy labels? Does it need training? Because in your instruction, you say use DAC to identify noisy labels and estimate the noisy label in the training dataset.

I guess I should use my training dataset to train DAC, and record the abstained sample. Then I remove this sample manually and retrain a DNN in standard cross entropy loss. Do I understand correctly?

**Comment by thulas** [comment]
@txw1997  correct. If you have an idea of how much noise is in your data, you can set the DAC abstention to that value.
